src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -104,7 +105,8 @@ public EventTypeService(final EventTypeRepository eventTypeRepository,
             eventType.setTopic(topicName);
             eventTypeRepository.saveEventType(eventType);
             return Result.ok();
-        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException e) {
+        } catch (final InvalidEventTypeException | NoSuchPartitionStrategyException |
+                DuplicatedEventTypeNameException e) {
```

Comment:
@antban makes sense. I changed it now in a way you suggested: https://github.com/zalando/nakadi/pull/615/commits/349f721502c1aa19d1947cb02e1913326572104e